sysfs-led-util.h

```diff
@@ -26,6 +27,53 @@
 
 # include <stdbool.h>
 
+/* ========================================================================= *
+ * UTILITY
+ * ========================================================================= */
+
+/** Clamp float value to given range
+ */
```

Comment:
This could be just single line comment like those below

Reply:
I guess the idea was to use this form but then something happened - lets go with the shorter version as you suggested.